workflow/scripts/h5.jl

```diff
@@ -0,0 +1,184 @@
+         (x, y) in zip(propdf.row_centroid, propdf.col_centroid)] for
+        c in ["latitude", "longitude"]
+    ]
+
+    x, y = [
+        [latlondata[c][z] for z in V] for
+        (c, V) in zip(["Y", "X"], [propdf.row_centroid, propdf.col_centroid])
+    ]
+
+    propdf.latitude = latitude
+    propdf.longitude = longitude
+    propdf.x = x
+    propdf.y = y
+    dropcols!(propdf, colstodrop)
+    return nothing
+end
+
+function dropcols!(df, colstodrop)
+    select!(df, Not(colstodrop))
+    return nothing
+end
+
+function converttounits!(propdf, latlondata, colstodrop)
+    if nrow(propdf) == 0
+        dropcols!(propdf, colstodrop)
+        return nothing
+    end
+    convertcentroid!(propdf, latlondata, colstodrop)
+    x = latlondata["X"]
+    dx = abs(x[2] - x[1])
+    convertarea(area) = area * dx^2 / 1e6
+    convertlength(length) = length * dx / 1e3
+    propdf.area .= convertarea(propdf.area)
+    propdf.convex_area .= convertarea(propdf.convex_area)
+    propdf.minor_axis_length .= convertlength(propdf.minor_axis_length)
+    propdf.major_axis_length .= convertlength(propdf.major_axis_length)
+    propdf.perimeter .= convertlength(propdf.perimeter)
+    return nothing
+end
+
+@pyinclude(joinpath(@__DIR__, "latlon.py"))
+getlatlon = py"getlatlon"
+
+"""
+    makeh5file(pathtosampleimg, resdir)
+
+Package the results of the IceFloeTracker pipeline in `resdir` into individual HDF5 files in `resdir/hdf5-files`. 
+
+This function expects the following files to be present in `resdir`: `filenames.jls`, `passtimes.jls`, `segmented_floes.jls`, and `floe_props.jls`. These files are generated by the `IceFloeTracker` pipeline.
+
+# Arguments:
+
+  * `pathtosampleimg`: Path to a sample image. This is used to extract the coordinate reference system (CRS) and the latitude and longitude coordinates of the image pixels.
+  * `resdir`: Path to the directory containing the results of the IceFloeTracker pipeline.
+
+# File structure
+Each HDF5 file has the following structure:
+
+```
+    🗂️ HDF5.File: (read-only) YYYYmmddTHHMM.sat.labeled_image.250m.h5
+    ├─ 🏷️ contact
+    ├─ 🏷️ crs
+    ├─ 🏷️ fname_reflectance
+    ├─ 🏷️ fname_truecolor
+    ├─ 🏷️ iftversion
+    ├─ 🏷️ reference
+    ├─ 📂 floe_properties
+    │  ├─ 🏷️ Description of labeled_image
+    │  ├─ 🏷️ Description of properties
+    │  ├─ 🔢 column_names
+    │  ├─ 🔢 labeled_image
+    │  └─ 🔢 properties
+    └─ 📂 index
+    ├─ 🔢 latitude
+    ├─ 🔢 longitude
+    ├─ 🔢 time
+    ├─ 🔢 x
+    └─ 🔢 y
+```
+# The `floe_properties` and `index` group
+
+The `floe_properties` group contains a floe properties matrix `properties` for `labeled_image` and associated `column_names`.
+The 'index' group contains the spatial coordinates in the source image coordinate reference system (default NSIDC polar stereographic, meters) and geographic coordinates (latitude and longitude, decimal degrees). Estimated satellite overpass time 'time' is provided in Unix time (seconds since 1970-01-01 00:00 UTC).
```

Comment:
```suggestion
The 'index' group contains the spatial coordinates in the source image coordinate reference system (default NSIDC polar stereographic, meters) and geographic coordinates (latitude and longitude, decimal degrees). Estimated satellite overpass time 'time' is provided in Unix timestamp format (seconds since 1970-01-01 00:00 UTC).
```